qiskit/schemas/qobj_schema.json

```diff
@@ -87,6 +87,22 @@
             ],
```

Comment:
Lets bump the version as per @mtreinish comment and than we should be good to go.

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4677/commits/aa84449b0258a5a817fefd056f19e69a382ba958